lib/elasticband/filter/range.rb

```diff
@@ -0,0 +1,29 @@
+module Elasticband
+  class Filter
+    class Range < Filter
+      attr_accessor :field, :ranges
+
+      RANGES = %i(gt gteq lt lteq time_zone)
```

Comment:
@rubensmabueno I do not understand the `time_zone` range. Could you add some info into the PR description?